src/JWT.Extensions.AspNetCore/Internal/DefaultIdentityFactory.cs

```diff
@@ -15,7 +15,7 @@ internal static class DefaultIdentityFactory
         public static IIdentity CreateIdentity(IDictionary<string, string> dic)
         {
```

Comment:
This method is not match with the signature of the IdentityFactory prop in JwtAuthenticationOptions.cs